app/assets/javascripts/components/articles/article.jsx

```diff
@@ -49,6 +49,15 @@ const Article = createReactClass({
     const formattedTitle = CourseUtils.formattedArticleTitle(this.props.article, this.props.course.home_wiki, this.props.wikidataLabel);
     const historyUrl = `${this.props.article.url}?action=history`;
 
+    let tracked;
+    if (this.props.current_user.isInstructor || this.props.current_user.admin) {
```

Comment:
I think just `current_user.isAdvancedRole` might work here as well.

Reply:
Yup. It does. Thanks!